libs/documentation/src/lib/components/result-list/demos/basic/result-list-basic.component.ts

```diff
@@ -1,21 +1,61 @@
-import { Component, OnInit, ViewChild, AfterViewInit, ChangeDetectorRef } from '@angular/core';
+import { Component, ChangeDetectorRef } from '@angular/core';
 
 @Component({
-  templateUrl: './result-list-basic.component.html',
+  templateUrl: './result-list-basic.component.html'
 })
-
 export class ResultListBasic {
+  constructor(private change: ChangeDetectorRef) {}
```

Comment:
remove unused ChangeDetector and remove import

Reply:
This has been taken care of.

Reply:
Are you sure you're pushing your changes? I still see this in your branch